kratos/tests/cpp_tests/geometries/test_line_3d_2.cpp

```diff
@@ -298,7 +298,7 @@ namespace Testing {
         KRATOS_CHECK_NEAR(JacobianDeterminant, ExpectedJacobian, TOLERANCE);
     }
 
-    KRATOS_TEST_CASE_IN_SUITE(Line3D2ShapeFunctionsValues, KratosCoreGeometriesFastSuite) {
+    KRATOS_TEST_CASE_IN_SUITE(Line3D2ShapeFunctionValue, KratosCoreGeometriesFastSuite) {
         auto geom = GeneratePointsDiagonalLine3D2();
```

Comment:
I think this should return a basetype-pointer and not the one of the derived class
Otherwise problems related to inheritance are hidden